tests/components/canary/test_config_flow.py

```diff
@@ -106,7 +106,7 @@ async def test_user_form_single_instance_allowed(hass, canary_config_flow):
 async def test_options_flow(hass):
     """Test updating options."""
     with patch("homeassistant.components.canary.PLATFORMS", []):
-        entry = await init_integration(hass)
+        entry = await init_integration(hass, skip_entry_setup=True)
```

Comment:
We need to set up the entry to make sure the config flow is loaded.

Reply:
We should probably patch the library to avoid the I/O instead.

Reply:
Will patch out `Api` instead

`strace -f -e trace=connect pytest tests/components/canary/test_config_flow.py `

Confirmed no connections